models/gitcoin/ethereum/gitcoin_ethereum_schema.yml

```diff
@@ -0,0 +1,141 @@
+version: 2
+
+models:
+  - name: gitcoin_ethereum_contribution_dataset
+    meta:
+      blockchain: ethereum
+      project: gitcoin
+      contributors: 1chioku
+    config:
+      tags: ['ethereum', 'gitcoin', 'dataset']
+    description: >
+        description: Dataset of Gitcoin contributions on Ethereum.
+    columns:
+      - &txn_id
+        name: txn_id
+        description: "An internal ID for a contribution (off-chain)"
+        tests: 
+          - not_null
+      - &user_id
+        name: user_id
+        description: "An internal ID for a contributor (off-chain)"
+        tests: 
+          - not_null
+      - &address
+        name: address
+        description: "The wallet address used to contribute"
+        tests: 
+          - not_null
+      - &grant_id
+        name: grant_id
+        description: "The ID of a grant on gitcoin (off-chain)"
+        tests: 
+          - not_null
+      - &chain
+        name: chain
+        description: "The chain used for the contribution"
+        tests: 
+          - not_null
+      - &txn_hash
+        name: txn_hash
+        description: "The hash of the transaction"
+        tests: 
+          - not_null
+      - &network
```

Comment:
If all data is for mainnet, we may don't need to store this column.

Reply:
Same as above, I do not want to change the source data, even though that specific column is technically irrelevant. I will let the users of that table determine how they want to process that data.